<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Computing topological data</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonInputFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input SpatialGraph file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInputFile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSGtiff">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>371</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutputFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Output TIFF file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputFile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_delta">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_delta">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TIFF voxel spacing (um)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dmin">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_dmin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum vessel diameter (um)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOutputFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>outputFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonInputFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>inputFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSGtiff</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>SGtiffer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>outputFileSelecter()</slot>
  <slot>inputFileSelecter()</slot>
  <slot>SGtiffer()</slot>
 </slots>
</ui>
